src/test/java/org/sagebionetworks/web/unitclient/widget/entity/controller/EntityActionControllerImplTest.java

```diff
@@ -1980,32 +1978,34 @@ public void testNotFolderDeletionPrompt() {
 	}
 
 	@Test
-	public void testConfigureManageAccessRequirements() {

```

Comment:
The original test was unclear to me, so I split this into two new tests where I assume we were testing the behavior for an ACT member and a non-ACT member

Reply:
I think the test names need to be revisited.  It looks like the first test is when isACTActionAvailable is false (currently named testConfigureManageAccessRequirementsAsACTMember), and the second is when the ACT Actions are available (currently named testConfigureManageAccessRequirementsAsACTMemberAsNonACTMember)